.github/workflows/build.yml

```diff
@@ -4,8 +4,6 @@ on:
   create:
     tags:
   push:
-    branches:
-      - master
```

Comment:
why is this changed?

Reply:
We changed it so that pushes to all branches trigger CI checks.

Reply:
I see, sounds good.